resources/model/OpenStudio.idd

```diff
@@ -9913,6 +9913,12 @@ OS:AirLoopHVAC,
        \autosizable
        \units m3/s
        \default 0
+  N2, \field Design Return Air Flow Fraction of Supply Air Flow
+       \note The design return air flow rate as a fraction of supply air flow rate with no exhaust.
+       \note This can be used to model a pressurized system or set to zero to model a DOAS with no return flow.
+       \note Use ZoneAirMassFlowConservation to balance zone infiltration and mixing flows.
+       \minimum 0.0
+       \required-field
```

Comment:
New field.

Made it \requird-field and moved the E+ default of 1.0 to the Ctor.